### PR TITLE
Upgrade the English Document

### PR DESCRIPTION
**Pull request by youqingxiaozhua**
The English version of the get_started document is too old and may fuse the newbie as in issue #1231 .

### CODE REVIEW

File: docs/en/tutorials/getting_started_en.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ After preparing the configuration file, The training process can be started in t
...
-    -o use_gpu=False
+    -c ./ppcls/configs/quick_start/MobileNetV3_large_x1_0.yaml \
+    -o Arch.pretrained=False \
+    -o Global.device=gpu
 ```
 
-Among them, `-c` is used to specify the path of the configuration file, `-o` is used to specify the parameters needed to be modified or added, `-o pretrained_model=""` means to not using pre-trained models.
-`-o use_gpu=True` means to use GPU for training. If you want to use the CPU for training, you need to set `use_gpu` to `False`.
+Among them, `-c` is used to specify the path of the configuration file, `-o` is used to specify the parameters needed to be modified or added, `-o Arch.pretrained=False` means to not using pre-trained models.
+`-o Global.device=gpu` means to use GPU for training. If you want to use the CPU for training, you need to set `Global.device` to `False`.
```
Comment (TingquanGao) [inline docs/en/tutorials/getting_started_en.md]: -> you need to set `Global.device` to `cpu`.

File: docs/en/tutorials/getting_started_en.md [hunk 2]
```diff
@@ -54,12 +54,12 @@ After configuring the configuration file, you can finetune it by loading the pre
...
-    -c configs/quick_start/MobileNetV3_large_x1_0_finetune.yaml \
-    -o pretrained_model="./pretrained/MobileNetV3_large_x1_0_pretrained" \
-    -o use_gpu=True
+    -c ./ppcls/configs/quick_start/MobileNetV3_large_x1_0.yaml \
+    -o Arch.pretrained=True \
+    -o Global.device=gpu
 ```
 
-Among them, `-o pretrained_model` is used to set the address to load the pretrained weights. When using it, you need to replace it with your own pretrained weights' path, or you can modify the path directly in the configuration file.
+Among them, `-o Arch.pretrained` is used to set the address to load the pretrained weights. When using it, you need to replace it with your own pretrained weights' path, or you can modify the path directly in the configuration file.
```
Comment (TingquanGao) [inline docs/en/tutorials/getting_started_en.md]: -> You can also set it into `True` to use pretrained weights that trained in ImageNet1k.

File: docs/en/tutorials/getting_started_en.md [hunk 3]
```diff
@@ -204,72 +188,57 @@ About parameter description, see [1.4 Model evaluation](#14-model-evaluation) fo
...
+    -c ./ppcls/configs/quick_start/MobileNetV3_large_x1_0.yaml \
+    -o Global.pretrained_model=output/MobileNetV3_large_x1_0/best_model
 ```
 
-Among them, the `--model` parameter is used to specify the model name, `--pretrained_model` parameter is used to specify the model file path, the path does not need to include the model file suffix name, and `--output_path` is used to specify the storage path of the converted model, class_dim means number of class for the model, default as 1000.
-
-**Note**:
-1. If `--output_path=./inference`, then three files will be generated in the folder `inference`, they are `inference.pdiparams`, `inference.pdmodel` and `inference.pdiparams.info`.
-2. You can specify the `shape` of the model input image by setting the parameter `--img_size`, the default is `224`, which means the shape of input image is `224*224`. If you want to use `Transformer series models`, such as `DeiT_***_384`, `ViT_***_384`, you need to set `--img_size=384`.
+Among them,  `Global.pretrained_model` parameter is used to specify the model file path.
```
Comment (TingquanGao) [inline docs/en/tutorials/getting_started_en.md]: -> Among them, `Global.pretrained_model` parameter is used to specify the model file path that does not need to include the file suffix name.

File: docs/en/tutorials/getting_started_en.md [hunk 4]
```diff
@@ -204,72 +188,57 @@ About parameter description, see [1.4 Model evaluation](#14-model-evaluation) fo
...
 
 The above command will generate the model structure file (`inference.pdmodel`) and the model weight file (`inference.pdiparams`), and then the inference engine can be used for inference:
 
+Go to the deploy directory:
+
+```
+cd deploy
+```
+
+Execute the command to inference. Cause the default value of `class_id_map_file` is the mapping file of the ImageNet dataset, we set it to `None` here.
```
Comment (TingquanGao) [inline docs/en/tutorials/getting_started_en.md]: -> Using inference engine to inference. Because the mapping file of ImageNet1k dataset is used by default, we should set `PostProcess.Topk.class_id_map_file` into `None`.

File: docs/en/tutorials/getting_started_en.md [hunk 5]
```diff
@@ -204,72 +188,57 @@ About parameter description, see [1.4 Model evaluation](#14-model-evaluation) fo
...
-+ `enable_mkldnn`: Wheter to use `MKL-DNN`, default by `False`. When both `use_gpu` and `enable_mkldnn` are set to `True`, GPU is used to run and `enable_mkldnn` will be ignored.
-+ `resize_short`: The length of the shortest side of the image that be scaled proportionally, default by `256`;
-+ `resize`: The side length of the image that be center cropped from resize_shorted image, default by `224`;
-+ `enable_calc_topk`: Whether to calculate top-k accuracy of the predction, default by `False`. Top-k accuracy will be printed out when set as `True`.
-+ `gt_label_path`: Image name and label file, used when `enable_calc_topk` is `True` to get image list and labels.
++ `Global.infer_imgs`: The path of the image file to be predicted;
++ `Global.inference_model_dir`: Model structure file path, such as `../inference/inference.pdmodel`;
++ `Global.use_tensorrt`: Whether to use the TesorRT, default by `False`;
++ `Global.use_gpu`: Whether to use the GPU, default by `True`
++ `Global.enable_mkldnn`: Wheter to use `MKL-DNN`, default by `False`. When both `Global.use_gpu` and `enable_mkldnn` are set to `True`, GPU is used to run and `enable_mkldnn` will be ignored.
```
Comment (TingquanGao) [inline docs/en/tutorials/getting_started_en.md]: -> `Global.enable_mkldnn`: Wheter to use `MKL-DNN`, default by `False`. It is valid when `Global.use_gpu` is be set into `False`.

**Review by TingquanGao — APPROVED** [review]
LGTM
